### ISSUE TITLE
Install Loop

### ISSUE THREAD

**Issue by BShurilla** [jshridha/docker-blueiris#7]
Was able to do an initial install just fine, although I was getting the same web error that was mentioned in Issue #2, - https://i.imgur.com/Pju78Y8.png - so then I restarted the container, and now it just continually just tries to run repair/modify/remove instead of actually launching blueiris.  Tried running through both repair, and modify, but it just loops back around into the same wizard.

**Comment by aaronstaves** [comment]
Same, just running the container as specified from the docs, also seeing a restart loop just trying to use the demo:

```
/usr/lib/python2.7/dist-packages/supervisor/options.py:297: UserWarning: Supervisord is running as root and it is searching for its configuration file in default locations (including its current working directory); you probably want to specify a "-c" argument specifying an absolute path to a configuration file for improved security.
  'Supervisord is running as root and it is searching '
2019-07-22 00:02:40,295 CRIT Supervisor running as root (no user in config file)
2019-07-22 00:02:40,295 WARN Included extra file "/etc/supervisor/conf.d/supervisord.conf" during parsing
2019-07-22 00:02:40,303 INFO RPC interface 'supervisor' initialized
2019-07-22 00:02:40,303 CRIT Server 'unix_http_server' running without any HTTP authentication checking
2019-07-22 00:02:40,303 INFO supervisord started with pid 1
2019-07-22 00:02:41,307 INFO spawned: 'x11vnc' with pid 13
2019-07-22 00:02:41,310 INFO spawned: 'X11' with pid 14
2019-07-22 00:02:41,312 INFO spawned: 'progman' with pid 15
2019-07-22 00:02:41,313 INFO spawned: 'x11vnc-loc' with pid 16
2019-07-22 00:02:41,315 INFO spawned: 'novnc' with pid 17
2019-07-22 00:02:41,529 INFO reaped unknown pid 42
2019-07-22 00:02:42,115 INFO reaped unknown pid 80
2019-07-22 00:02:42,116 INFO reaped unknown pid 46
2019-07-22 00:02:42,183 INFO reaped unknown pid 78
2019-07-22 00:02:42,320 INFO success: x11vnc entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2019-07-22 00:02:42,321 INFO success: X11 entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2019-07-22 00:02:42,321 INFO success: progman entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2019-07-22 00:02:42,321 INFO success: x11vnc-loc entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2019-07-22 00:02:42,321 INFO success: novnc entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2019-07-22 00:02:59,601 INFO exited: progman (exit status 0; expected)
2019-07-22 00:03:00,019 INFO spawned: 'progman' with pid 85
2019-07-22 00:03:01,118 INFO success: progman entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2019-07-22 00:03:08,714 INFO exited: progman (exit status 0; expected)
2019-07-22 00:03:09,718 INFO spawned: 'progman' with pid 90
2019-07-22 00:03:09,723 INFO reaped unknown pid 59
```

**Comment by aaronstaves** [comment]
Nevermind, I must've had an old install in the same director I was using as the volume mount.  I cleared that out to get a fresh install and now looks like it's up and running.  

This might help you @BShurilla

**Comment by aaronstaves** [comment]
Well, nevermind again.  Looks like BI updated to version 5 and the Dockerfile needs to be upgraded.

tl;dr it installs fine but:

1. it's looking for BlueIris4 to be installed
2. doesn't find it, installs BlueIris5
3. re-runs script, which should now find the newly installed BlueIris5
4. goto 1

I have most of a fix here https://github.com/jshridha/docker-blueiris/pull/8, but the **initial** install gets in a loop.  Once it's installed and you restart the container, it seems to work fine.

**Comment by jshridha** [comment]
This should be fixed now. Thanks again for the help.
